help/submit.md

```diff
@@ -11,7 +11,7 @@ Submission Guidelines
 -   Authors must agree to the [submission terms and agreement](policies/submission_agreement) and [Instructions for submission](policies/instructions_for_submission).
 
 -   Authors are expected to self-submit. Submissions by a third party are accepted under limited conditions. See instructions for [third-party submissions](third_party_submission) and [index submissions](submit_index) for conference proceedings.
--   New submissions received by 14:00 (Eastern Daylight/Standard Time Zone) are generally made [available](/help/versions) at 20:00 (Eastern) but are subject to delays in the moderation process (see the [schedule for details](#availability).) 
+-   New submissions received by 14:00 (Eastern Daylight/Standard Time Zone) are generally made [available](/help/versions) at 20:00 (Eastern) based on the [schedule for for availability](availability).

```

Comment:
the "available" link is not very accessibility-friendly because that word on it's own doesnt convey the intent of the link. But I am not sure what to suggest instead! Maybe...
```suggestion
-   New submissions received by 14:00 (Eastern Daylight/Standard Time Zone) are generally made available at 20:00 (Eastern) based on the [schedule for for availability](availability). Also see [versions help pages](/help/versions)

```